packages/rocketchat-ui/client/views/app/createChannel.js

```diff
@@ -164,6 +170,9 @@ Template.createChannel.events({
 	},
 	'change [name="type"]'(e, t) {
 		t.type.set(e.target.checked ? e.target.value : 'd');
+		if (!e.target.check) {
```

Comment:
How does this reflect the comment? I don't see a validation with respect to public channels only. And afterall: should this not be `checked`?